<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgTB</class>
 <widget class="QDialog" name="DlgTB">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="GLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="LabelNumber">
       <property name="text">
        <string>Number:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="EditNumber">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="LabelReleaseDate">
       <property name="text">
        <string>Release date:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QDateEdit" name="EditReleaseDate">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="LabelTitle">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="EditTitle">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="LabelRegisteredBy">
       <property name="text">
        <string>Registered by:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="EditRegisteredBy">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="LabelCategory">
       <property name="text">
        <string>Category:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="ComboCategory">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="LabelReplaces">
       <property name="text">
        <string>Replaces:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="EditReplaces">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="LabelRK">
       <property name="text">
        <string>RK:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="EditRK">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="LabelReplacedBy">
       <property name="text">
        <string>Replaced by:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="EditReplacedBy">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="LabelTechPub">
       <property name="text">
        <string>Tech Pub:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="EditTechPub">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="LabelComment">
       <property name="text">
        <string>Comment:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="3">
      <widget class="QPlainTextEdit" name="TexteditComment">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="tabChangesFocus">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="LabelKeywords">
       <property name="text">
        <string>Keywords:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QLineEdit" name="EditKeywords">
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="HLayoutButtons">
     <item>
      <spacer name="HSpacerButtons">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonWebPage">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Visit web page</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonDownloadRM">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Download RM</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>EditNumber</tabstop>
  <tabstop>EditTitle</tabstop>
  <tabstop>ComboCategory</tabstop>
  <tabstop>EditRK</tabstop>
  <tabstop>EditTechPub</tabstop>
  <tabstop>EditReleaseDate</tabstop>
  <tabstop>EditRegisteredBy</tabstop>
  <tabstop>EditReplaces</tabstop>
  <tabstop>EditReplacedBy</tabstop>
  <tabstop>TexteditComment</tabstop>
  <tabstop>EditKeywords</tabstop>
  <tabstop>ButtonOK</tabstop>
  <tabstop>ButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
